### PR TITLE
Create comprehensive list of useful queries

### PR DESCRIPTION
**Pull request by RobertNugent04**
Created all of the useful queries for the database in a SQL file which cover the following areas:

- Basic Queries
- Update-related Queries
- Licence Management Queries
- Users and Access Management Queries
- Machine Monitoring and Status Queries
- Package Version Queries
- Action Log and Auditing Queries

Examples from the SQL file:
usefulq

### CODE REVIEW

**Review by kevashans — DISMISSED** [review]
Tables might change in the future, especially about licensing. this needs further discussion.  a good start just to get our feet wet and thinking about what type of queries will be used

File: upflux_db.sql [hunk 1]
```diff
@@ -0,0 +1,334 @@
+USE upflux;
```
Comment (PatrickOrjieh) [inline upflux_db.sql:3]: Can you add in the create database statement at the top but you can comment it out

Comment (RobertNugent04) [reply upflux_db.sql:3]: Thanks for the feedback. I have added that now.

**Review by PatrickOrjieh — APPROVED** [review]
Good job

**Review by kevashans — APPROVED** [review]

**Pull request merged**